src/main/java/seedu/address/model/event/StartDate.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.model.event;
+
+public class StartDate {
```

Comment:
Is it possible to add a constructor with some parameters instead of just an empty class?